server/channels/app/post_test.go

```diff
@@ -326,11 +326,12 @@ func TestUpdatePostInArchivedChannel(t *testing.T) {
 
 	archivedChannel := th.CreateChannel(th.Context, th.BasicTeam)
 	post := th.CreatePost(archivedChannel)
-	th.App.DeleteChannel(th.Context, archivedChannel, "")
+	deleteErr := th.App.DeleteChannel(th.Context, archivedChannel, "")
+	require.Nil(t, deleteErr)
 
-	_, err := th.App.UpdatePost(th.Context, post, true)
-	require.NotNil(t, err)
-	require.Equal(t, "api.post.update_post.can_not_update_post_in_deleted.error", err.Id)
+	_, appErr := th.App.UpdatePost(th.Context, post, true)
+	require.NotNil(t, appErr)
+	require.Equal(t, "api.post.update_post.can_not_update_post_in_deleted.error", deleteErr.Id)
```

Comment:
Copy past error :wink: 
```suggestion
	require.Equal(t, "api.post.update_post.can_not_update_post_in_deleted.error", appErr.Id)
```